<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionUpload_flow_data"/>
    <addaction name="actionDownload_Weather_Data"/>
    <addaction name="actionDownload_GeoTiff"/>
    <addaction name="actionUniformize_Flow_and_Rain"/>
    <addaction name="actionRead_Weather_Data"/>
    <addaction name="actionSelect_Area"/>
   </widget>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpload_flow_data">
   <property name="text">
    <string>Download flow data</string>
   </property>
  </action>
  <action name="actionDownload_Weather_Data">
   <property name="text">
    <string>Download Weather Data</string>
   </property>
  </action>
  <action name="actionDownload_GeoTiff">
   <property name="text">
    <string>Download GeoTiff</string>
   </property>
  </action>
  <action name="actionUniformize_Flow_and_Rain">
   <property name="text">
    <string>Uniformize Flow and Rain</string>
   </property>
  </action>
  <action name="actionRead_Weather_Data">
   <property name="text">
    <string>Read Weather Data</string>
   </property>
  </action>
  <action name="actionSelect_Area">
   <property name="text">
    <string>Select Area</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
